core/src/lib/auth/providers/anonymous/AnonymousCredential.ts

```diff
@@ -24,7 +24,7 @@ export default class AnonymousCredential implements StitchCredential {
    */
   public readonly providerCapabilities = new ProviderCapabilities(true);
 
-  constructor(providerName: string) {
+  constructor(providerName: string = ProviderTypes.ANON) {
```

Comment:
[minor] This should really be changed to match java where we have a DEFAULT_NAME and TYPE value inside the *AuthProvider file itself. The type is technically not a name.